alpine/Dockerfile.tpl

```diff
@@ -4,9 +4,24 @@ FROM #{FROM}
 
 #{QEMU}
 
-RUN apk add --update bash udev dbus tar ca-certificates \
+RUN apk add --update bash udev dbus tar ca-certificates openrc \
 	&& rm -rf /var/cache/apk/*
 
+# Config OpenRC
+RUN echo 'rc_sys="lxc"' >> /etc/rc.conf \
+	&& echo 'rc_provide="loopback net"' >> /etc/rc.conf \
+	&& echo 'rc_logger="YES"' >> /etc/rc.conf \
+	&& echo 'rc_env_allow="*"' >> /etc/rc.conf \
+	&& echo 'rc_crashed_start="YES"' >> /etc/rc.conf \
+	&& echo 'rc_crashed_stop="NO"' >> /etc/rc.conf \
```

Comment:
It makes sense to have `/etc/rc.conf` as a file in the repo and then `COPY` it in instead of echoing all the lines